deal.II-toolchain/packages/dealii.package

```diff
@@ -17,6 +17,7 @@ INSTALL_PATH=${ORIG_INSTALL_PATH}/deal.II-${VERSION}
 
 CONFOPTS=" \
 -D CMAKE_BUILD_TYPE=DebugRelease \
+-D DEAL_II_EARLY_DEPRECATIONS:BOOL=ON \
```

Comment:
I am not sure if we should set this by default. This is more of a developer option.